pkg/apis/cloudcredential/v1/types_credentialsrequest.go

```diff
@@ -53,6 +53,12 @@ type CredentialsRequestSpec struct {
 	// ProviderSpec contains the cloud provider specific credentials specification.
 	// +kubebuilder:pruning:PreserveUnknownFields
 	ProviderSpec *runtime.RawExtension `json:"providerSpec,omitempty"`
+
+	// ServiceAccountNames contains a list of service accounts that will use permissions granted by this
+	// CredentialsRequest. This is not used by CCO but instead needed for ccoctl tool to set up access
+	// control in the cloud provider.
```

Comment:
My suggestion is kind of wordy, but I'm trying to capture why this information is needed in a generic way...

```suggestion
	// ServiceAccountNames contains a list of ServiceAccounts that will use permissions associated with this
	// CredentialsRequest. This is not used by CCO, but the information is needed for being able to properly
	// set up access control in the cloud provider when the ServiceAccounts are used as part of the cloud
	// credentials flow.
```